README.md

```diff
@@ -298,3 +298,40 @@ jfrogClient.artifactory()
     console.error(error);
   });
 ```
+
+### Platform
+
+#### Web Login
+#### Register For Web Login
+To register for web login, use the following code:
+
+```javascript
+const sessionId = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX // UUID
+jfrogClient
+  .platform()
+  .WebLogin()
+  .registerSessionId(sessionId)
+  .then((result) => {
+    ...
+  })
+  .catch((error) => {
+    ...
+  });
+```
+#### Get Access Token From Web Login
```

Comment:
You can't make the title its parent to have the same number of #
```suggestion
##### Get Access Token From Web Login
```

Please also add the title to the table of contents